### ISSUE TITLE
Should intro/splashscreen be visible?

### ISSUE THREAD

**Issue by ST6Daniel** [Nibre/MotherVR#76]
Using WMR, I wonder if it is intended for us WMR users to see the intro logo? (20th century fox etc, I only hear it, but am still in the empty steam environment)

**Comment by Nibre** [comment]
No, this is just an issue with how SteamVR works with the game right now. It is intentional, as otherwise the intro videos get literally 'glued' to your eyes

**Comment by Nibre** [comment]
I'd also like to add, that for you or anyone else annoyed by this issue, until it's fixed, for the time being you can include `-skipintro` into the Launch Options in Steam, and it will skip the intro videos and jump straight into the main menu.

**Comment by NeonUniverse** [comment]
Could this also be a legacy vr bug? Could also be how the game renders the intro vids. IDK just an idea
